infrahub_sdk/schema/__init__.py

```diff
@@ -102,6 +110,14 @@ def validate_data_against_schema(self, schema: MainSchemaTypesAPI, data: dict) -
                     message=f"{key} is not a valid value for {identifier}",
                 )
 
+    def set_cache(self, schema: dict[str, Any] | BranchSchema, branch: str | None = None) -> None:
```

Comment:
I'd add a short docstring here to explain that you can use this to prepopulate a schema.